### ISSUE TITLE
Method in CommentSection wrong condition

### ISSUE THREAD

**Issue by naynyeinchan** [RatMD/bloghub-plugin#8]
Method need to be 
```
    protected function getFrontendUser()
    {
        if (PluginManager::instance()->hasPlugin('RainLab.User')) {
            $rainLabAuth = \RainLab\User\Classes\AuthManager::instance();
            if ($rainLabAuth->check()) {
                return $rainLabAuth->getUser();
            } else {
                return null;
            }
        } else {
            return null;
        }
    }
```

Instead of (when plugin is activated returning null)
```
    protected function getFrontendUser()
    {
        if (PluginManager::instance()->hasPlugin('RainLab.User')) {
            return null;
        }

        $rainLabAuth = \RainLab\User\Classes\AuthManager::instance();
        if ($rainLabAuth->check()) {
            return $rainLabAuth->getUser();
        } else {
            return null;
        }
    }
```

**Comment by SamBrishes** [comment]
Thanks for your bug reports. I'll try to push a new version today (or tomorrow).

~ Sam.

**Comment by SamBrishes** [comment]
Fixed in v1.3.3 (Version has been pushed to Octobers Marketplace).

**Comment by naynyeinchan** [comment]
Thanks for your prompt response and fix.
